### ISSUE TITLE
Tracker % Complete Error

### ISSUE THREAD

**Issue by serkanyazicioglu** [Questie/Questie#5632]
## Bug description
Checking on SoD with latest version. When Tracker objective sorting is set to % Complete it throws following error

Message: Interface/AddOns/Questie/Localization/l10n.lua:132: invalid option in `format'
Time: Wed Feb  7 23:05:44 2024
Count: 1
Stack: Interface/AddOns/Questie/Localization/l10n.lua:132: invalid option in `format'
[string "=[C]"]: ?
[string "@Interface/AddOns/Questie/Localization/l10n.lua"]:132: in function <Interface/AddOns/Questie/Localization/l10n.lua:122>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/Questie/Modules/Tracker/QuestieTracker.lua"]:657: in function `_UpdateQuests'
[string "@Interface/AddOns/Questie/Modules/Tracker/QuestieTracker.lua"]:1632: in function `Update'
[string "@Interface/AddOns/Questie/Modules/Tracker/QuestieTracker.lua"]:257: in function `func'
[string "@Interface/AddOns/Questie/Modules/Libs/QuestieCombatQueue.lua"]:25: in function <...e/AddOns/Questie/Modules/Libs/QuestieCombatQueue.lua:17>

Locals: (*temporary) = "Quests (By % Complete)"

**Comment by MyTechnoHunter** [comment]
Can we get some more information please and thank you Game language and screenshots

**Comment by serkanyazicioglu** [comment]
I'm playing on SoD at the moment and UI language is English. Here's a screenshot of current quests when I switch sorting to by zone. I've quests from 3 different zones: Redridge Mountains, Stormwind City, The Deadmines

![image](https://github.com/Questie/Questie/assets/3519644/4e325828-7676-42a5-98f7-8470eb2143af)

![image](https://github.com/Questie/Questie/assets/3519644/c3aa6e46-9031-44d0-94ee-ecd3c88e73c1)

**Comment by serkanyazicioglu** [comment]
Also this error is getting thrown regardless of the zone quest(s) being tracked. I've reduced tracked quest to one and tried with each zone separately. Switching to 'By % Complete' throws error for all of the zones I have.

![image](https://github.com/Questie/Questie/assets/3519644/e3af7623-cc22-4c82-8801-a91c71659922)

Latest screenshot mentions AI_VoiceOver but I disabled that addon and tried again, this time error mentions a different addon.
